Objects/codeobject.c

```diff
@@ -1630,15 +1630,10 @@ deopt_code(PyCodeObject *code, _Py_CODEUNIT *instructions)
 {
     Py_ssize_t len = Py_SIZE(code);
     for (int i = 0; i < len; i++) {
-        int opcode = _Py_GetBaseOpcode(code, i);
-        if (opcode == ENTER_EXECUTOR) {
-            _PyExecutorObject *exec = code->co_executors->executors[instructions[i].op.arg];
-            opcode = _PyOpcode_Deopt[exec->vm_data.opcode];
-            instructions[i].op.arg = exec->vm_data.oparg;
-        }
-        assert(opcode != ENTER_EXECUTOR);
-        int caches = _PyOpcode_Caches[opcode];
-        instructions[i].op.code = opcode;
+        _Py_CODEUNIT inst = _Py_GetBaseCodeUnit(code, i);
+        assert(inst.op.code <= RESUME);
```

Comment:
What is the meaning of `opcode <= RESUME` (what is this category of opcodes)? Can we have a macro for this to make it clear?